tests/functional/context_methods/test_builtin_functions.py

```diff
@@ -105,26 +109,18 @@ def test_builtin_invocation_args_dict_function(self, project):
 
         parsed_logs = parse_json_logs(log_output)
         result = find_result_in_parsed_logs(parsed_logs, "invocation_result")
-
         assert result
-
-        # Result is checked in two parts because profiles_dir is unique each test run
-        expected = "invocation_result: {'debug': True, 'log_format': 'json', 'write_json': True, 'use_colors': True, 'printer_width': 80, 'version_check': True, 'partial_parse': True, 'static_parser': True, 'profiles_dir': "
-        assert expected in str(result)
```

Comment:
I don't think we really need this extra check (it's a dictionary of values, inside a string). But I could see rewriting this test to check for specific values from within the Jinja context, e.g.
```
macros__validate_invocation_sql = """
{% macro validate_invocation(my_variable) %}
    {{ log("use_colors: "~ invocation_args_dict['use_colors']) }}
{% endmacro %}
"""
```
ok I've convinced myself, I'm going to do that now